src/main/resources/static/public/javascript/account.js

```diff
@@ -1,5 +1,12 @@
 $(document).ready(function () {
 
+    $("#inputCountry").click(function() {
+            country = $( "#inputCountry option:selected" ).attr("value");
+
+            var newcode = $( "#inputCountry option:selected" ).attr("code");
```

Comment:
Maakt voor nu geen zak uit hoor, maar consistentie in camelCasing voorkomt vaak problemen